examples/neural_compressor/question-answering/README.md

```diff
@@ -83,3 +83,18 @@ and [pruning](https://github.com/huggingface/optimum/blob/main/examples/config/p
 configuration files will be used.
 
 The flag `--verify_loading` can be passed along to verify that the resulting quantized model can be loaded correctly.
+
+The distillation process can be accelerated by reserving teacher logits.
```

Comment:
I would be in favor of adding `generate_teacher_logits` to the already existing command line presenting knowledge distillation for user to use it by default and to not have too many examples as well (to have a good readability)